draft-bbs-signatures.md

```diff
@@ -59,13 +59,15 @@ BBS is a digital signature scheme categorized as a form of short group signature
 
+The name BBS is derived from the authors of the original academic works of Dan Boneh, Xavier Boyen and Hovav Schacham, where the scheme was first described.
+
 Beyond the core properties of a digital signature scheme, BBS signatures provide multiple additional unique properties, three key ones are:
 
 **Selective Disclosure** - The scheme allows a signer (issuer) to sign multiple messages and produce a single -constant size- output signature. An intermediary (prover) then possessing the messages and the signature can generate a proof whereby they can choose which messages to disclose, while leaking no-information about the un-disclosed messages. The proof itself guarantees the integrity and authenticity of the disclosed messages (e.g. that they were originally signed by the issuer).
 
-**Unlinkable Proof Presentations** - The proofs generated by the scheme are known as zero-knowledge, proofs-of-knowledge of the signature, meaning a verifying party in receipt of a proof is unable to determine which signature was used to generate the proof, removing a common source of correlation. In general each proof generated is indistinguishable from random even for two proofs generated from the same signature.
+**Unlinkable Proofs** - The proofs generated by the scheme are known as zero-knowledge, proofs-of-knowledge of the signature, meaning a verifying party in receipt of a proof is unable to determine which signature was used to generate the proof, removing a common source of correlation. In general, each proof generated is indistinguishable from random even for two proofs generated from the same signature.
 
-**Proof of Possession** - The proofs generated by the scheme prove to a verifier that the party who generated the proof (prover) was in possession of a signature without revealing it. The scheme also supports binding a presentation header to the generated proof. The presentation header can include arbitrary information such as a cryptographic nonce, an audience/domain identifier to ensure the generated proof can only be used appropriately, including providing a way for a verifier to detect a replay attack, or any other information a protocol may require to be cryptographically tied to a proof.
+**Proof of Possession** - The proofs generated by the scheme prove to a verifier that the party who generated the proof (prover) was in possession of a signature without revealing it. The scheme also supports binding a presentation header to the generated proof. The presentation header can include arbitrary information such as a cryptographic nonce, an audience/domain identifier and or time based validity information.
```

Comment:
```suggestion
**Proof of Possession** - The proofs generated by the scheme prove to a verifier that the party who generated the proof (prover) was in possession of a signature without revealing it. The scheme also supports binding a presentation header to the generated proof. The presentation header can include arbitrary information such as a cryptographic nonce, an audience/domain identifier and/or time based validity information.
```
Maybe^?? Or remove the "or"??

Reply:
`/` often implies or so adding it would be and or or IMO?

Reply:
I've rapidly checked: and/or is used as an "inclusive or" because saying "or" might be inclusive or exclusive. I think we could use it then but feel free to discard this suggestion!